content/en/tracing/api_catalog/_index.md

```diff
@@ -1,18 +1,94 @@
+- Select a combination of facets on the left.
+
+You can [define tags](#adding-bulk-tags) to use as facets so you can easily find groups of endpoints you're most interested in.
+
+**Note**: The `API Type` facet is based on auto-generated tags with values of `public` and `unknown` (which usually indicates a private endpoint).
+
+Use the **Group by** selector to group endpoints together by opinionated tags, so you can better navigate a large collection of endpoints in the list. The list group shows aggregated performance data for the endpoints in the group.
+
+## Investigating endpoint details
+
+Click an endpoint in the list to open a details page that shows performance, governance, and metadata information for the endpoint, collected from various areas of Datadog into one place. Here you can edit the short name and description, and add custom tags. You can also launch deeper investigations of the various types of telemetry using links into other areas of Datadog. 
+
+For example, click on the monitors section to open a detailed card with more information and a link to monitors for the endpoint.
+
+The performance graphs on the page are initially scoped to the same settings as on the API Catalog page, and you can change those settings in the details page  to suit your investigation by using the time frame selector and other scope drop-downs. 
+
+## Adding bulk tags
+
+In addition to tagging an endpoint in its details page, you can **add tags to multiple endpoints** at once in the API Catalog. Select multiple endpoint check boxes, and click **Edit tags** to provide ownership, importance, or other useful grouping information labels to the selected endpoints. This can help you quickly access groups of endpoints defined by your own criteria, and create assets such as monitors and dashboards from endpoints with specific tags.
+
+For example, if you want to create latency alerts for endpoints that are particularly sensitive to performance problems, tag those endpoints with a tag like `Latency sensitive`. Or tag endpoints that handle sensitive data with a tag like `PII` and alert when endpoints with that tag have a `401 Unauthorized` response. Other examples of useful tags:
+
+- Criticality level
+- `experimental`
+- `contains password`
+- `contains PII`
+- Business logic (for example: `payments` )
+- Specific customer group (`subscribers`)
+
+When you add a tag, it appears in the list of facets on the left of the catalog. Click a facet to filter the list and add the tag to the Search query field.
+
+## Exploring endpoints during incidents
+
+When investigating an incident, getting a higher resolution view through the API endpoints can lead to faster understanding of the root cause. Resolve incidents faster with **API ownership information** -- team, code repository, on-call details -- directly associated with each endpoint, to know who to reach when something goes wrong. 
```

Comment:
let's replace this paragraph with:

When investigating an incident, getting a detailed resolution view of the API endpoint can lead to faster understanding of the root cause.

In addition, you can quickly identify who is the owning *Team* of each endpoint, who is the *on-call engineer* and *how to reach them* (Email, Slack, Pager Duty) to help resolve incidents involving a specific API Endpoint faster.

*we don't have the navigation to code yet in this version